taipy/core/_orchestrator/_dispatcher/_job_dispatcher.py

```diff
@@ -61,7 +61,7 @@ def stop(self, wait: bool = True, timeout: Optional[float] = None):
             self.join(timeout=timeout)
 
     def run(self):
-        self._logger.info("Start job dispatcher...")
+        self._logger.debug("Start job dispatcher...")
```

Comment:
What do you think?
```suggestion
        self._logger.debug("Job dispatcher started...")
```

Reply:
I think may be `Job dispatcher starting...`  is better?

Reply:
Yeah, i think `Job dispatcher started...` is better. thanks! What confuse me is the `...`  and ` started` seems not suitable